### PR TITLE
- added a wipe policy function

### PR DESCRIPTION
**Pull request by aleph2012**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@ghaiklor @nodece @zhmushan @Sefriol please review.

File: API.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Implements a policy adapter for casbin with MongoDB support.
     * _instance_
         * [.setFiltered([isFiltered])](#MongooseAdapter+setFiltered)
         * [.loadPolicyLine(line, model)](#MongooseAdapter+loadPolicyLine)
+        * [.wipePolicy()](#MongooseAdapter+wipePolicy) ⇒ <code>Promise.&lt;void&gt;</code>
```
Comment (hsluoyz) [inline API.md:13]: We should use `clearPolicy()` name.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "casbin-mongoose-adapter",
-  "version": "2.1.0",
+  "name": "@beamnetwork/casbin-mongoose-adapter",
```
Comment (hsluoyz) [inline package.json:2]: Not OK.

**Comment by onichandame** [comment]
Just wondering what would happen currently when `Enforcer.clearPolicy` is invoked. Would it delete all policies in the adapter?

**Comment by Sefriol** [comment]
It clears the policy from the enforcer, but not from the adapter.

**Pull request closed without merging**